File: src/server/TypeORM/src/entity/agile_system.ts
import {Entity, PrimaryGeneratedColumn, Column} from "typeorm";

@Entity()
export class agile_system {

    @PrimaryGeneratedColumn()
    agile_system_id: string;

    @Column()
    agile_system_name: string;

    @Column()
    agile_system_type: string;

    @Column()
    work_team_id: string;

}

File: fg_metrics/VSTS/VSTSComms/VSTSComms/EntitiesMySQL/AgileSystem.cs
using System;
using System.Collections.Generic;

namespace VSTSComms.EntitiesMySQL
{
    public partial class AgileSystem
    {
        public AgileSystem()
        {
            AgileSprint = new HashSet<AgileSprint>();
        }

        public string AgileSystemId { get; set; }
        public string AgileSystemName { get; set; }
        public string AgileSystemType { get; set; }
        public string WorkTeamId { get; set; }
        public bool? Active { get; set; }

        public WorkTeam WorkTeam { get; set; }
        public ICollection<AgileSprint> AgileSprint { get; set; }
    }
}
